src/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from "react";
+import { Platform, StyleSheet, Text, View } from "react-native";
+
+const isSignedIn = () =>
+  // mock function for the authentication check
+  Promise.resolve(false);
+const instructions = Platform.select({
+  ios: "Press Cmd+R to reload,\nCmd+D or shake for dev menu",
+  android:
+    "Double tap R on your keyboard to reload,\n" +
+    "Shake or press menu button for dev menu"
+});
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
We will be using BEM for class naming?

Reply:
react-native does not have any css, so naming is not an issue